test/e2e/cypress/integration/profiles/mfa/totp.spec.ts

```diff
@@ -292,7 +292,7 @@ context("2FA TOTP", () => {
 
       // The React app keeps using the same flow. The following scenario used to be broken,
       // because the internal context wasn't populated properly in the flow after settings were saved.
-      it.only("should allow changing other settings and then setting up totp", () => {
+      it("should allow changing other settings and then setting up totp", () => {
```

Comment:
Looks like this was forgotten [a few weeks ago](https://github.com/ory/kratos/pull/2960/files#diff-154448811eddc216f531ef50d9a613a5c6c2f0bf544f9896f8d45617e8a68ed2R295). These tests are still passing, though. So crisis averted. 

Would be great to have an ESLint rule or some mechanism to prevent `.only`s ending up on master. But AFAICT, there is nothing of that sort for cypress :(